### ISSUE TITLE
UK Stores

### ISSUE THREAD

**Issue by elypsis** [Yorisoft/puppet-purchase#10]
This project looks really interesting.

Do you plan on including any UK stores in the future? Such as Curry's, Amazon-uk, argos and buyer?

Thanks in advance

**Comment by Yorisoft** [comment]
Thanks @elypsis ! And I appreciate your interest ! 

The tldr is, yes but will take some time to get to. Not sure about Amazon tho. 

Long Answer:

Currently I'm working on optimizing the workflow for all current stores, which is a priority. For transparency sake, I can tell you that currently the top priorities are :
    a. Optimize the workflow of all bots 
        - fix steps that aren't working, speed up checkout, mainly things to make them more reliable while unattended.
    b. Fix the CICD pipeline that I've created for the project: 
        - Having a lot of issues running test using jenkins/docker atm. 

I can definitely start working on adding UK stores once I get done with the above. 

As for Amazon, that's a bit trickier. The problem being retail stores like Amazon that allow others to sell on their site. It could cause the bot to end up buying an item for the user but from someone other than the major retailer and possible at a higher price than the user was expecting to pay. 
The issue can be circumvented if the site has a specific URL for their own listing - like Walmart. Walmart has unique URLs that distinguish between Walmart sold items and third-party sold items. My understanding is that Amazon bunches them all together.

**Comment by elypsis** [comment]
thanks @Yorisoft
